crates/sovereign-sdk/rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -63,7 +67,11 @@ pub trait ZkvmHost: Zkvm + Clone {
 /// Must support recursive proofs.
 pub trait Zkvm: Send + Sync {
     /// A commitment to the zkVM program which is being proven
-    type CodeCommitment: Clone + Debug + Serialize + DeserializeOwned;
+    type CodeCommitment: Clone
+        + Debug
+        + Serialize
+        + DeserializeOwned
+        + Into<risc0_zkvm::sha::Digest>;
```

Comment:
again here also, let's use something like Into<[u8; 32]>

Reply:
If we don't expose the receipt, this won't be needed, else, will update this.

Reply:
no longer relevant with the current version